package.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "@dpc-sdp/ripple",
-  "version": "1.0.0-beta.1",
```

Comment:
Removed it as it won't be updated by Lerna. Babel didn't has that in their monorepo as well.